src/renderer/gdi/lib/gdi.vcxproj

```diff
@@ -22,6 +22,11 @@
     <ClInclude Include="..\gdirenderer.hpp" />
     <ClInclude Include="..\precomp.h" />
   </ItemGroup>
+  <ItemDefinitionGroup>
+    <Lib>
+      <AdditionalDependencies>usp10.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
as above, you can safely remove this as conhost/gdi is windows 8.1+ only

Reply:
We do need this dependency because otherwise the linker complains that it can't find the Script* Unicode symbols. The docs only warn that the import order is apparently important on older OS.

Reply:
I see. In that case, you must make sure that any sources file that consumes RendererGdi is updated to link with this library as well

Reply:
Specifically the "sources" and "dirs" files for the Windows nmake build. It will be very important to ensure those work after this change.